test/remote/ip_blocklist_tests.js

```diff
@@ -0,0 +1,106 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+var test = require('tap').test
+var Promise = require('bluebird')
+var restify = Promise.promisifyAll(require('restify'))
+var TestServer = require('../test_server')
+var mcHelper = require('../memcache-helper')
+
+var TEST_EMAIL = 'test@example.com'
+var ACTION = 'dummyAction'
+var BLOCK_IP = '1.93.0.224'
+var BLOCK_IP_INRANGE = '0.1.0.0'
+var VALID_IP = '3.0.0.0'
+
+process.env.IP_BLOCKLIST_ENABLE = true
+process.env.IP_BLOCKLIST_LOGONLY = false
+process.env.IP_BLOCKLIST_FILES = [
+  './../test/mocks/firehol_level1.netset',
+  './../test/mocks/firehol_level2.netset',
+  './../test/mocks/firehol_level3.netset',
+  './../test/mocks/firehol_level4.netset',
+  './../test/mocks/firehol_webserver.netset',
```

Comment:
I'm not sure there's much value in committing *all* of these files to the repo here, especially the big ones.  Would it suffice to include only the leve1 list, for syntax-compatibility-testing purposes?